### PR TITLE
Lang fixes

### PR DESCRIPTION
**Pull request by ravenbuilder934**
Closes #
Closes #
Closes #

# Changes proposed in this pull request:
- Updates en_us.json to fix some grammar and match 1.15 more closely
-
-

Review please

### CODE REVIEW

File: src/main/resources/assets/structurize/lang/en_us.json [hunk 1]
```diff
@@ -153,7 +153,7 @@
   "block.structurize.gated_acacia_paper_timber_frame": "Gate Framed Acacia Paper Timber Frame",
   "block.structurize.up_gated_jungle_dark_oak_timber_frame": "Up Gate Framed Jungle Dark Oak Timber Frame",
   "block.structurize.blockbarreldeco_onside": "Barrel Horizontal",
-  "block.structurize.birch_blockpaperwall": "Birch Paperwall",
```
Comment (Nightenom) [inline src/main/resources/assets/structurize/lang/en_us.json:156]: I would say this is generated by datagen

Comment (ravenbuilder934) [reply src/main/resources/assets/structurize/lang/en_us.json:156]: The name of the item is generated by datagen? Can it still be changed?

Comment (Nightenom) [reply src/main/resources/assets/structurize/lang/en_us.json:156]: Blame Ash, he hasn't made paper walls datagen. But yes, timber frames translations are generated

**Review by Nightenom — APPROVED** [review]

File: src/main/resources/assets/structurize/lang/en_us.json (hunk 1 above)
Comment (ravenbuilder934) [reply src/main/resources/assets/structurize/lang/en_us.json:156]: Good thing I didn't change any of the timber frame names then

**Pull request merged**